packages/cli/src/commands/start.ts

```diff
@@ -60,7 +60,7 @@ export class Start extends BaseCommand {
 		}),
 	};
 
-	protected activeWorkflowRunner = Container.get(ActiveWorkflowRunner);
```

Comment:
This was getting called before `super.init();`, which in turn was creating an instance of the `NodeTypes` service before we were done loading nodes and credentials. 
This is not an issue for `nodes-base` as those nodes get loaded lazily. But, community nodes that are loaded right away, ended up not getting the proper post-processing, which mean they were missing polling related functionality.